cmd/tink-server/main.go

```diff
@@ -32,7 +33,28 @@ func main() {
 	errCh := make(chan error, 2)
 	facility := os.Getenv("FACILITY")
 
-	cert, modT := rpcServer.SetupGRPC(ctx, logger, facility, errCh)
+	tinkDB := db.Connect(logger)
+
+	_, onlyMigration := os.LookupEnv("ONLY_MIGRATION")
+	if onlyMigration == true {
+		logger.Info("Applying migrations. This process will end when migrations will take place.")
+		numAppliedMigrations, err := tinkDB.Migrate()
+		if err != nil {
+			panic(err)
```

Comment:
```
log.Fatal(err)
panic(err)
```
so we can get a proper error message logged + rollbar if configured, don't really need the panic but is a good signal to send to the programmer.

Reply:
This
> rollbar if configured

and [this](https://github.com/tinkerbell/tink/pull/296#discussion_r492291406) do not play nice together, you can pick only one. You can have rollback or not I presume.